Makefile

```diff
@@ -85,6 +85,10 @@ e2e: node_modules test-resetdb
 quicktest:
 		go test -failfast -short -tags sqlite ./...
 
+.PHONY: quicktest-hsm
+quicktest-hsm:
```

Comment:
This builds the docker image but does not execute tests?

Reply:
Actually it does by specifying `--target test-hsm`
`docker build --progress=plain -f .docker/Dockerfile-hsm --target test-hsm .`
that will execute https://github.com/ory/hydra/pull/2625/files#diff-ae54bef08e3587b28ad8e93eb253a9a5cd9ea6f4251977e35b88dc6b42329e25R20-R38

Reply:
Ah got it, that is some arcane Docker wizzardry :D